concepticondata/conceptlists/Oliveira-2014-517.tsv

```diff
@@ -0,0 +1,518 @@
+Oliveira-2014-517-447	447	raiz	root		670	ROOT		§444
+Oliveira-2014-517-448	448	pau velho, pau podre	old stick, rotten stick					§445
+Oliveira-2014-517-449	449	rodar	to run		1519	RUN		§446
+Oliveira-2014-517-450	450	roupa	clothes		1895	CLOTHES		§447
+Oliveira-2014-517-451	451	tipo de cesto	type of cedar					§448
+Oliveira-2014-517-452	452	sal	salt		1274	SALT		§449
+Oliveira-2014-517-453	453	pauzada (amontoado de tronco que desce no rio)	branches that hang into the river					§450
+Oliveira-2014-517-454	454	taboca utilizada para fazer flechas	bamboo used for making arrows					§451
+Oliveira-2014-517-455	455	quebrar	to break		680	BREAK (DESTROY OR GET DESTROYED)		§452
+Oliveira-2014-517-456	456	zarabatana	blowgun		432	BLOWGUN		§453
+Oliveira-2014-517-457	457	broto	sprout		3358	SPROUT (SHOOT)		§454
+Oliveira-2014-517-458	458	pescoço	neck		1333	NECK		§455
+Oliveira-2014-517-459	459	esp. de gavião	species of hawk					§456
+Oliveira-2014-517-460	460	pomo de adão	adam's apple		802	ADAM'S APPLE		§457
+Oliveira-2014-517-461	461	nominalizador de instrumento	nominalizer for instruments					§458
+Oliveira-2014-517-462	462	bater, golpear	to hit, to strike		1433	HIT		§459
+Oliveira-2014-517-463	463	espingarda	shotgun		3948	SHOTGUN		§460
+Oliveira-2014-517-464	464	coador	colander					§461
+Oliveira-2014-517-465	465	engravidar, dar a luz	to impregnate, to give birth				GIVE BIRTH	§462
+Oliveira-2014-517-466	466	esp. de rã ou sapo	species of frog					§463
+Oliveira-2014-517-467	467	azul escuro	dark blue					§464
+Oliveira-2014-517-468	468	verruga	wart		2665	WART		§465
+Oliveira-2014-517-469	469	fazer	to do		813	DO		§466
+Oliveira-2014-517-470	470	fazer	to do		813	DO		§467
+Oliveira-2014-517-471	471	água, rio	water, river				WATER//RIVER	§468
+Oliveira-2014-517-472	472	esp. de peixe	species of fish					§469
+Oliveira-2014-517-473	473	pupunha	peach palm		3919	PEACH PALM		§470
+Oliveira-2014-517-474	474	esp. de abóbora	species of pumpkin					§471
+Oliveira-2014-517-475	475	esp. de macaco	species of monkey					§472
+Oliveira-2014-517-476	476	varrer, ciscar	to sweep		230	SWEEP		§473
+Oliveira-2014-517-477	477	erva	grass		606	GRASS		§474
+Oliveira-2014-517-478	478	teia de aranha	spider web		1065	SPIDER WEB		§475
+Oliveira-2014-517-479	479	algodão	cotton		1850	COTTON		§476
+Oliveira-2014-517-480	480	sufixo verbal imperativo	verbal imperative suffix					§477
+Oliveira-2014-517-481	481	igarapé (?)	igarapé					§478
+Oliveira-2014-517-482	482	outro	other		197	OTHER		§479
+Oliveira-2014-517-483	483	um (numeral)	one		1493	ONE		§480
+Oliveira-2014-517-484	484	canela, parte inferior da perna	cinnamon, lower part of leg				LOWER LEG	§481
+Oliveira-2014-517-485	485	cheiro	smell		3704	ODOR (SMELL)		§482
+Oliveira-2014-517-486	486	remar	to row		1849	ROW		§483a
+Oliveira-2014-517-487	487	remo	oar		3137	OAR		§483b
+Oliveira-2014-517-488	488	remo	oar		3137	OAR		§484
+Oliveira-2014-517-489	489	borduna, bastão	stick		1295	STICK		§485
```

Comment:
Borduna is more like a piece of wood used as a weapon, bastão stick should be 'walking stick'. I don't think that stick captures both meanings accurately.